ext/ftp/ftp.stub.php

```diff
@@ -2,133 +2,62 @@
 
 /** @generate-function-entries */
 
-/** @return resource|false */
+/** @return \FTPConnection|false */
```

Comment:
The return type was defined as a PHPDoc type hint only because of `resource`. It makes a perfect return type declaration after your changes :)

Reply:
Done.